### ISSUE TITLE
N900: New IM in conversation shows no contact

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#268]
```
What steps will reproduce the problem?
1. set availability online 
2. all msn contacts captured correctly in contact book
3. however in conversation it shows no contact in the new IM tab

What is the expected output? What do you see instead?
in conversation, when select new IM, expect all off line and online contacts to 
be visible, and can start IM conversation from there.

What version of the product are you using? On what operating system?
n900, 10.2010.19-1.002, msn (pecan) protocol plugin 0.1.0-0maemo3 from maemo.org

Please provide any additional information below.
tried uninstall and reinstall with the same problem. manual rebooted after 
installation.
firmware update OTA. only install pecan after firmware update. did not perform 
any contact restore from backup since don;t have any contact problem after OTA.
previously have skype setup and can see off line and online contacts in the new 
IM in conversation.
after installing msn pecan, skype contact now also not visible. (i.e.: new IM 
shows no contact).
```

Original issue reported on code.google.com by `barnabas...@gmail.com` on 8 Jun 2010 at 3:32

**Comment by GoogleCodeExporter** [comment]
```
So, I go to conversations, and I see the old conversations and I click one of 
the online contacts and just type the new message. Is that what you can't do? 
(I can't see any "New IM" action)

You also said you don't have any problem after OTA update, so when do you have 
the problem?
```

Original comment by `felipe.contreras` on 8 Jun 2010 at 8:29

**Comment by GoogleCodeExporter** [comment]
```
Ahh, I didn't notice the "New IM" button at the top, I can reproduce the 
problem but I don't think it's msn-pecan's fault.
```

Original comment by `felipe.contreras` on 9 Jun 2010 at 10:54
- Changed title: **N900: New IM in conversation shows no contact**
- Changed state: **Accepted**
- Added labels: **Type-Defect**, **OpSys-Maemo**

**Comment by GoogleCodeExporter** [comment]
```
Jonny Lamb found the root cause of the problem: missing Capabilities interface 
in telepathy-haze.

It should still not affect other CMs (GTalk, Skype), that's a bug in the 
Conversations UI:
https://bugs.maemo.org/show_bug.cgi?id=10621
```

Original comment by `felipe.contreras` on 9 Jun 2010 at 12:17
- Added labels: **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
I checked my google talk and skype contact, sometimes they appear and sometimes 
not. I could not really replicate the problem. most probably has to do with 
dropped connection, or switching from wifi to 3g etc. So maybe it is indeed not 
msn pecan problem but maemo5 conversation ui issue?
```

Original comment by `barnabas...@gmail.com` on 10 Jun 2010 at 4:26

**Comment by GoogleCodeExporter** [comment]
```
Yes, it's a conversation ui issue, see c3.
```

Original comment by `felipe.contreras` on 10 Jun 2010 at 9:01

**Comment by GoogleCodeExporter** [comment]
```
There's a new version of telepathy-haze that implements the Capabilities 
interface. I've updated telepathy-msn-pecan to the latest haze and included 
this patch.

Fixed in telepathy-msn-pecan 0.1.0-0maemo4.
```

Original comment by `felipe.contreras` on 12 Jun 2010 at 4:32
- Changed state: **Fixed**
